### PR TITLE
Fix intent scheme links.

### PR DESCRIPTION
**Pull request by IhmeHippi**
Also added flag for verbose intent link error dialogs.

### CODE REVIEW

File: sdk/src/main/java/com/swedbankpay/mobilesdk/internal/WebViewFragment.kt [hunk 1]
```diff
@@ -99,6 +111,86 @@ internal class WebViewFragment : Fragment() {
         return true
     }
 
+    private fun handleIntentUri(uri: Uri) {
+        try {
+            val intent = getIntentUriIntent(uri)
+            val context = requireContext()
+            try {
+                context.startActivity(intent)
+            } catch (e: Exception) {
```
Comment (payex-dev) [inline sdk/src/main/java/com/swedbankpay/mobilesdk/internal/WebViewFragment.kt:120]: Codacy found an issue: [TooGenericExceptionCaught - [e] at /src/sdk/src/main/java/com/swedbankpay/mobilesdk/internal/WebViewFragment.kt:120:22](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-android/pullRequest?prid=6032614)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-android/pullRequest?prid=6032614&bid=19769640)

File: sdk/src/main/java/com/swedbankpay/mobilesdk/internal/WebViewFragment.kt [hunk 2]
```diff
@@ -193,13 +281,41 @@ internal class WebViewFragment : Fragment() {
...
+        cause: Throwable?
+    ) : Exception(message, cause)
+
+    class IntentUriErrorDialogFragment : DialogFragment() {
         companion object {
             const val ARG_URI = "U"
+            const val ARG_EXTRA_MESSAGE = "E"
 
-            fun newInstance(uri: Uri) = ErrorDialogFragment().apply {
+            fun newInstance(exception: IntentUriException, verbose: Boolean) = IntentUriErrorDialogFragment().apply {
```
Comment (payex-dev) [inline sdk/src/main/java/com/swedbankpay/mobilesdk/internal/WebViewFragment.kt:295]: Codacy found an issue: [UndocumentedPublicFunction - [newInstance] at /src/sdk/src/main/java/com/swedbankpay/mobilesdk/internal/WebViewFragment.kt:295:17](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-android/pullRequest?prid=6032614)

Comment (IhmeHippi) [reply sdk/src/main/java/com/swedbankpay/mobilesdk/internal/WebViewFragment.kt:295]: Codacy apparently does not understand internal functions (or classes, etc.) properly. This is not a public function.

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-android/pullRequest?prid=6032614&bid=19769640)

**Pull request merged**